### DISCUSSION TITLE
Driver version for rtd1619b

### DISCUSSION THREAD

**Discussion by Galokurt** [bb-qq/r8152#356] [Q&A] [answered]
Hello everyone
I wanted to know if there was a version of the driver developed for rtd1619b (NAS DS223, DS223j, DS423, DS124...)?

**Comment by bb-qq** [comment, marked as answer]
here
https://github.com/bb-qq/r8152/releases/download/2.19.2-1/r8152-rtd1619b-2.19.2-1_7.2.spk

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
